docs/api.rst

```diff
@@ -942,6 +956,22 @@ Timeshift a :ref:`Stream` by *n* milliseconds. ::
 
 Delaying a :ref:`Stream` timeshifts all the events by the same amount. It doesn't change the time *between* events.
 
+.. _withLocalTime:
+
+withLocalTime
+`````````````
+
+.. code-block:: haskell
+
+  withLocalTime :: Offset -> Stream a -> Stream a
+
+Create a Stream with localized :ref:`Time` values, whose origin (i.e., time 0) is at the specified Time on the :ref:`Scheduler` used to the Stream is observed with :ref:`runEffects` or :ref:`run`.
+
+When implementing custom higher-order :ref:`Stream` combinators, such as :ref:`chain`, you should use ``withLocalTime`` to localize "inner" Streams before running them.
```

Comment:
How is `chain` a custom combinator that the user implements?

Reply:
"such as" is referring to chain as a higher-order combinator.  Would you suggest removing the word "custom"?